cfg/system.config.in

```diff
@@ -117,3 +118,17 @@ ffi-lib-dir       = @FFILibDir@
 #=======================
 
 with-libdw = @UseLibdw@
+
+# Installation:
+#=======================
+
+prefix          = @prefix@
```

Comment:
Well this is a good point, although `prefix` is already a commonly used term, but it is a bit confusing in itself. So `install-prefix` is probably better